packages/client/.depcheckrc

```diff
@@ -0,0 +1,4 @@
+ignores: [
```

Comment:
could those extends the main one ?

Reply:
I think since none of them share the same ignores, there's no point in having a base one. We don't want to ignore things that don't exist in a sub-package in case someone accidentally adds it later

Reply:
Ok, guess it make sense.
I was asking the question because the DevX team suggested that for our ts-config or linter-config.